### PR TITLE
6375 migrate role part2

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
gravitee-io/issues#6375

**Description**

Next step for role migration screen

**Additional context**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-kyxmszhsfz.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6375-migrate-role-part2/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-roles.component.html [hunk 1]
```diff
@@ -77,6 +77,6 @@ <h2 gioTableOfContents>{{ item.scope }}</h2>
   <gio-table-of-contents
     class="org-settings-roles__toc"
     [sectionNames]="{ '': 'Roles' }"
-    scrollingContainer="ng-roles-component"
+    scrollingContainer="ng-roles"
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-roles.component.html:80]: 😘

Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-roles.component.html:80]: :aa:

File: gravitee-apim-console-webui/src/organization/configuration/roles/role/org-settings-role.component.ts [hunk 2]
```diff
@@ -124,3 +144,14 @@ export class OrgSettingsRoleComponent implements OnInit, OnDestroy {
       });
   }
 }
+
+const fromFormPermissionToPermission = (formPermissions: Record<string, Record<'C' | 'R' | 'U' | 'D', boolean>>): Role['permissions'] => {
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/roles/role/org-settings-role.component.ts]: fromFormPermissionsToPermissions

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-owwsxxifup.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6375-migrate-role-part2/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-kdsblimggh.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6375-migrate-role-part2/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**